### PR TITLE
fix: Look at both rights and license to find dataset's terms of use

### PR DESCRIPTION
**Pull request by bellisk**

### CODE REVIEW

File: ckanext/switzerland/helpers/frontend_helpers.py [hunk 1]
```diff
@@ -146,47 +146,6 @@ def get_political_level(political_level):
     return political_levels.get(political_level, political_level)
 
 
-def get_terms_of_use_icon(terms_of_use):
```
Comment (bellisk) [inline ckanext/switzerland/helpers/frontend_helpers.py:149]: Removed this helper as it is no longer used anywhere.

File: ckanext/switzerland/helpers/frontend_helpers.py [hunk 2]
```diff
@@ -146,47 +146,6 @@ def get_political_level(political_level):
...
-        ogdch_term_utils.TERMS_OF_USE_CLOSED: {
-            'title': _('Closed data'),
-            'icon': 'terms_closed',
-        },
-    }
-    term_id = ogdch_term_utils.simplify_terms_of_use(terms_of_use)
-    return term_to_image_mapping.get(term_id, None)
-
-
-def get_terms_of_use_url(terms_of_use):
```
Comment (bellisk) [inline ckanext/switzerland/helpers/frontend_helpers.py:176]: Removed this helper as it is no longer used anywhere.

File: ckanext/switzerland/helpers/frontend_helpers.py [hunk 3]
```diff
@@ -146,47 +146,6 @@ def get_political_level(political_level):
...
-    terms_of_use_url = url_for('/terms-of-use')
-    pagemark = mapping_terms_of_use_to_pagemark.get(terms_of_use)
-    if pagemark:
-        terms_of_use_url += pagemark
-    return terms_of_use_url
-
-
-def get_dataset_terms_of_use(pkg):
```
Comment (bellisk) [inline ckanext/switzerland/helpers/frontend_helpers.py:184]: Removed this helper as the new helper `terms_of_use_utils.get_dataset_terms_of_use` does the same thing but more simply.

File: ckanext/switzerland/tests/test_helpers_terms_of_use.py [hunk 4]
```diff
@@ -1,17 +1,149 @@
...
+                        {"rights": "A very cool set of rights"},
+                    ]
+                },
+                ogdch_term_utils.TERMS_OF_USE_CLOSED
+            ],
+            [
+                {
+                    "resources": [
+                        {
+                            "license": "A very cool open license",
```
Comment (stefina) [inline ckanext/switzerland/tests/test_helpers_terms_of_use.py:103]: 😎

**Review by stefina — APPROVED** [review]

**Pull request merged**